syntax = "proto3";

option java_multiple_files = true;
package com.kugmax.learn.digitgateway.digitgateway.services;

message  DigitRecognitionResponse {
    int32 value = 1;
}

message DigitRecognitionByPathRequest {
    string path = 1;
}

message DigitRecognitionByBytesRequest {
    string bytes = 1;
}

service DigitRecognition {
    rpc RecognizeByPath(DigitRecognitionByPathRequest) returns (DigitRecognitionResponse) {}
    rpc RecognizeByBytes(DigitRecognitionByBytesRequest) returns (DigitRecognitionResponse) {}
}